src/components/BoardView/BoardView.js

```diff
@@ -0,0 +1,43 @@
+import React, { Component } from "react";
+import styled from "styled-components";
+
+import { amount, randomNr } from "../../helpers";
+
```

Comment:
Redundant newlines between import statements.

Reply:
This newline has been used to separate dependencies from the import of code written by me.